src/main/java/com/parkhomovsky/bookstore/controller/BookController.java

```diff
@@ -21,38 +24,45 @@
 @RequiredArgsConstructor
 @RestController
 @RequestMapping(value = "/books")
+@Tag(name = "Book management", description = "Endpoints for managing products")
 public class BookController {
     private final BookService bookService;
 
     @PutMapping("/{id}")
     public BookDto update(
-            @PathVariable Long id, @RequestBody CreateBookRequestDto createBookRequestDto) {
+            @PathVariable Long id, @RequestBody @Valid CreateBookRequestDto createBookRequestDto) {
         return bookService.update(id, createBookRequestDto);
     }
 
     @PostMapping
     @ResponseStatus(HttpStatus.CREATED)
+    @Operation(summary = "Create new book", description = "Adding new book to database")
     public BookDto createBook(@RequestBody @Valid CreateBookRequestDto book) {
         return bookService.create(book);
     }
 
     @GetMapping("/{id}")
+    @Operation(summary = "Get book by id", description = "Try to find book with provided id")
```

Comment:
As for me, "Try to find book with provided id" sounds strange. "Find" logic on repository layer, on service and controller layer only "get". And "try" also looks not ok.